CHANGELOG.md

```diff
@@ -138,7 +138,15 @@ Bugfixes and minor improvements:
 - Provide more information about the service on /version endpoint (service name, commit ID)
 - API base url (e.g. the one provided to CLI) is normalized
 - Fixes for CLI
+- Make version endpoint type consistent
+- Fix passing args to sequence through CLI
```

Comment:
Let's be consistent in writing the words Instance and Sequence in the descriptions using capital letters.

Reply:
fixed